sdk/core/azure-core/azure/core/tracing/ext/opencensus_wrapper.py

```diff
@@ -71,11 +44,12 @@ def span_instance(self):
         return self._span_instance
 
     def span(self, name="child_span"):
-        # type: (str) -> OpencensusSpanWrapper
+        # type: (Optional[str]) -> OpenCensusSpan
         """
         Create a child span for the current span and append it to the child spans list in the span instance.
         :param name: Name of the child span
-        :return: The OpencensusWrapper that is wrapping the child span instance
+        :type name: str
+        :return: The OpenCensusWrapper that is wrapping the child span instance
```

Comment:
`OpenCensusWrapper` doesn't exist. Probably ought to rename the file as well.

Reply:
done

Reply:
Actually for some reason I do not feel good about renaming the file.

Reply:
I renamed the file